src/dashboards/components/RefreshContext.tsx

```diff
@@ -31,21 +35,24 @@ export interface AutoRefreshState {
 }
 
 const jumpAheadTime = () => {
-  return moment()
-    .add(moment.duration(DEFAULT_TIME_AHEAD))
-    .format('YYYY-MM-DD HH:mm:ss')
+  const newTime = incrementDate(new Date(), DEFAULT_TIME_AHEAD, 'h')
+
+  const formatter = createDateTimeFormatter('YYYY-MM-DD HH:mm:ss')
+  return formatter.format(newTime)
 }
 
 const calculateTimeout = (timeout: string, timeoutUnit: string) => {
   const timeoutNumber = parseInt(timeout, 10)
-  const startTime = moment(new Date())
-  const copyStart = startTime.unix()
-  const endTime = startTime.add(
-    timeoutNumber as any,
+  const startTime = new Date()
+  const copyStart = startTime.getTime()
+  const endTime = incrementDate(
+    startTime,
+    timeoutNumber,
     timeoutUnit[0].toLowerCase()
   )
-  const cutoff = endTime.unix() - copyStart
-  return cutoff * 1000
+  const cutoff = endTime.getTime() - copyStart
```

Comment:
:+1: nice. This way doesn't lose millisecond precision like getting the time in seconds than multiplying by 1000 does.